.github/workflows/ci.yml

```
@@ -0,0 +1,98 @@
+name: CI
+
+on:
+  pull_request: ~
+  push:
+    branches:
+      - master
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: Ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php: ['7.1', '7.2', '7.3', '7.4', '8.0']
+        composer: ['']
+        phpunit: ['']
+        deprecation: ['']
+        coverage: ['none']
+        symfony: ['']
+        stability: ['']
+        include:
+          # Minimum supported dependencies with the latest and oldest PHP version
+          - php: 8.0
+            composer: --prefer-stable --prefer-lowest
+            deprecation: max[direct]=0
+          - php: 7.1
+            composer: --prefer-stable --prefer-lowest
+            deprecation: max[direct]=0
+
+          # coverage
+          - php: 8.0
+            phpunit: '--coverage-text'
+            coverage: 'xdebug'
+
+          # symfony version
+          - php: 8.0
+            symfony: '^3.0'
+          - php: 8.0
+            symfony: '^4.0'
+          - php: 8.0
+            symfony: '^5.0'
+
+          # dev
+          - php: 8.0
+            stability: 'dev'
+
+    steps:
+      - name: Set up PHP
+        uses: shivammathur/setup-php@2.7.0
+        with:
+          php-version: ${{ matrix.php }}
+          coverage: ${{ matrix.coverage }}
```

Add this and we dont need to install symfony/flex ourselves. 

```suggestion
          coverage: ${{ matrix.coverage }}
          tools: flex
```